.github/workflows/create-release.yaml

```diff
@@ -2,7 +2,7 @@ name: "Create release"
 
 env:
   IMAGE_REPO: europe-docker.pkg.dev/kyma-project/prod/kyma-environment-broker
-#  KYMA_BTP_MANAGER_REPO: ${{ github.repository_owner }}/btp-manager
+  KYMA_REPO: ${{ github.repository_owner }}/kyma-environment-broker
```

Comment:
Maybe we could rename `KYMA_REPO` to something more KEB related.

Reply:
deleted, not used

Reply:
We need an env like this to be able to use is as an env variable in the scripts.